config/initializers/active_admin.rb

```diff
@@ -189,7 +189,7 @@
   #   config.register_stylesheet 'my_print_stylesheet.css', media: :print
   #
   # To load a javascript file:
-  #   config.register_javascript 'my_javascript.js'
+  config.register_javascript 'ckeditor/init.js'
```

Comment:
Just so that you know, there is another way to load the JS. You could add

``` coffeescript
#= require ckeditor/init
```

to app/assets/javascripts/active_admin.js.coffee

I guess using the initializer is "cleaner" (seems like it's ActiveAdmin's recommandation), but it also moves the responsibility of loading a JS asset away from Rails' asset pipeline, so it'll be easier to forget about that ckeditor/init.js file being loaded. Both ways are fine I guess :)